docs/developers_guide/cnvmap_struct_c.md

```diff
@@ -0,0 +1,94 @@
+<!--
+(C) copyright 2020 VT SuperDARN, Virginia Polytechnic Institute & State University
+author: Kevin Sterne
+-->
+
+# Structure in C for the Convection Map data
+
+## Summary
+
+A description of the data structure in the C language used to store the convection map data.
+
+## Description
+
+### Introduction
```

Comment:
```suggestion
```
The introduction seems redundant but that is a style preference.

Reply:
I could see that, it was a copy over from the RFC style. I think it gives a little better to the subsection headings for the structs that follow maybe?